Tests/FoundationTests/DateExtensionsTests.swift

```diff
@@ -275,9 +275,9 @@ final class DateExtensionsTests: XCTestCase {
 		XCTAssertEqual(date.isInWeekend, Calendar.current.isDateInWeekend(date))
 	}
 	
-	func testIsInWeekday() {
+	func testisWorkday() {
```

Comment:
@kaphacius Could you please rename this to `testIsWorkday` to keep consistency with other tests :)